src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c

```diff
@@ -1493,7 +1493,16 @@ exec_execute_write_cmd(struct exec_core *exec, struct xocl_cmd* xcmd)
 static int
 exec_execute_copybo_cmd(struct exec_core *exec, struct xocl_cmd* xcmd)
 {
-	return 1; // error for now
+	int ret;
+	struct ert_start_copybo_cmd *ecmd =
```

Comment:
Do you want to add ert_start_copybo_cmd to the union under xocl_cmd?  I think consolidating the various supported command formats under xocl_cmd is good